docs/source/advanced_usage.rst

```diff
@@ -390,5 +358,3 @@ alpha.
     print(parametric_prog(1.0))
     print(parametric_prog(2.0))
```

Comment:
Should we change the name `parametric_prog` so that users searching for Quil 2.0's parametric program don't get sent here?

Reply:
+1

Reply:
done